src/Ztm.Data.Entity/Contexts/Main/Input.cs

```diff
@@ -41,5 +41,35 @@ public int CompareTo(Input other)
 
             return 0;
         }
+
+        public override bool Equals(Object other)
+        {
+            if (other == null)
+            {
+                return false;
+            }
+
+            if (GetType() != other.GetType())
+            {
+                return false;
+            }
+
+            return CompareTo((Input)other) == 0;
+        }
+
+        public override int GetHashCode()
+        {
+            int hash = 0;
+
+            hash ^= (TransactionHash != null) ? TransactionHash.GetHashCode() : 0;
+            hash ^= (int)Index;
```

Comment:
After discussion we decided to turn off this LGTM alert and use `Index.GetHashCode()` instead.